sdcm/sct_config.py

```diff
@@ -1775,6 +1785,18 @@ def __init__(self):
                 raise ValueError("For PasswordAuthenticator authenticator authenticator_user and authenticator_password"
                                  " have to be provided")
 
+        # 13) validate stress and prepare duration:
+        if stress_duration := self.get('stress_duration'):
+            try:
+                self['stress_duration'] = abs(int(stress_duration))
+            except ValueError:
+                raise ValueError(f'Configured stress duration for generic test duratinon have to be \
+                                 positive integer number in minutes. Current value: {stress_duration}') from ValueError
+            try:
+                self['prepare_duration'] = abs(int(self.get('prepare_duration')))
+            except ValueError:
+                self['prepare_duration'] = 300  # 5h
```

Comment:
why do we have default here in the code ? if the value is set incorrect, we should fail.

Reply:
set raise exception